driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/access.bpf.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+#include <helpers/interfaces/variable_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(access_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct auxiliary_map *auxmap = auxmap__get();
+	if(!auxmap)
+	{
+		return 0;
+	}
+
+	auxmap__preload_event_header(auxmap, PPME_SYSCALL_ACCESS_E);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: mode (type: PT_UINT32) */
+	unsigned long mode = (u32)extract__syscall_argument(regs, 1);
```

Comment:
Looks like you read the mode instead of the pathname here.

Reply:
Yeah this was to mirror the behavior of the `access_e` [probe](https://github.com/falcosecurity/libs/blob/c1d075ffda41dcbdeec0a9fee86f288b7b360d19/driver/bpf/fillers.h#L1046):

```c
FILLER(sys_access_e, true)
{
	unsigned long val;
	int res;

	/*
	 * mode
	 */
	val = bpf_syscall_get_argument(data, 1);
	res = bpf_val_to_ring(data, access_flags_to_scap(val));

	return res;
}
```

Reply:
Apologies for the extra commits. I've rebased and the unrelated changes are now gone.